src/main/java/com/ecommerce/wlm_ecommerce_product_service/domain/exception/ProductWasNotUpdated.java

```diff
@@ -0,0 +1,7 @@
+package com.ecommerce.wlm_ecommerce_product_service.domain.exception;
+
+public class ProductWasNotUpdated extends RuntimeException {
```

Comment:
Usually a custom exceptions must be created to be catch and some additional work can be done. If you extend RuntimeException - it is not checked exception and it will not be catched. So i don't see a reason why you need this exception at all